ethereumj-core/src/main/java/org/ethereum/net/eth/handler/Eth62.java

```diff
@@ -453,8 +454,15 @@ protected synchronized void processBlockHeaders(BlockHeadersMessage msg) {
     }
 
     protected synchronized void processGetBlockBodies(GetBlockBodiesMessage msg) {
-        List<byte[]> bodies = blockchain.getListOfBodiesByHashes(msg.getBlockHashes());
-
+        Iterator<byte[]> bodiesIterator = blockchain.getIteratorOfBodiesByHashes(msg.getBlockHashes());
+        List<byte[]> bodies = new ArrayList<>();
+        int sizeSum = 80; // ArrayList skeleton
```

Comment:
i) it's a magic number ii) does `80` bytes matter much with regard to `32Kb` limit?